server/app/controllers/api/v1/programminglanguages_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Api::V1::ProgramminglanguagesController < ApplicationController
```

Comment:
minor: is it possible to instead have the URL like programming-languages i.e. kebab case? it is a bit more readable and is more or less standard these days, I think